be/src/io/cache/block_file_cache.cpp

```diff
@@ -562,7 +561,7 @@ std::string BlockFileCache::clear_file_cache_async() {
 
 void BlockFileCache::recycle_deleted_blocks() {
     using namespace std::chrono;
-    static int remove_batch = 100;
+    static int remove_batch = 500;
```

Comment:
keep it 100, and make the `cond.wait_for(cache_lock, std::chrono::microseconds(100));` configurable to do throttle.